src/status_im/ui/screens/browser/views.cljs

```diff
@@ -102,6 +105,26 @@
        :accessibility-label :modal-chat-button}
       [icons/icon :main-icons/message]]]))
 
+(defn show-access-request [resources on-allow on-deny]
+  (utils/show-confirmation {:title               (i18n/label :t/permission-request)
```

Comment:
Consider that window.title can be reset, spoofed, might not be unique and can be reused by other pages if the permission is persistent, same goes window.location.